### ISSUE TITLE
glGenBuffers

### ISSUE THREAD

**Issue by NotLopX** [defold/editor2-issues#2392]
![capture](https://user-images.githubusercontent.com/16010495/50935846-0f0e6500-1433-11e9-9cab-075e6a774ba2.PNG)
My Gpu is Intel(R) G41 Express Chipset (Microsoft Corporation - WDDM 1.1)
I try downloading Windows SDK and the mt.exe wasn't there even when I downloaded Visual studio it wasn't there

**Comment by erikangelin-king** [comment]
Hi! Please try with the patched [java.exe](https://github.com/defold/editor2-issues/files/2054343/java.zip).

**Comment by erikangelin-king** [comment]
Just replace the java.exe found in Defold\packages\jre\bin with the patched one.

**Comment by NotLopX** [comment]
It’s not starting up now

**Comment by NotLopX** [comment]
What do I do now?

**Comment by erikangelin-king** [comment]
Can help further tomorrow.

**Comment by NotLopX** [comment]
Yes I’m using Windows 10. I’ve already tried updating my graphics driver. And yes nothing happens when I start it, defold will not run

**Comment by NotLopX** [comment]
Are you there?

**Comment by erikangelin-king** [comment]
Hi again @NotLopX,
To get more information it would be great if you could start Defold from the command line, and log the output. To do that, first go to `Help>About` and copy the two SHA strings listed. The top SHA string - for instance `ec036ca54d6e8062e0c567a8c3e238f12b1b744a` - identifies the current editor version, and is part of the `.jar` file name in the command line below. The bottom SHA string - for instance `8def4b50c9aa049670e698d7dff3dc00d77e42d5`-  identifies the current engine version, and is used when building native extensions. It is also part of the command line below. Next, Open `cmd.exe` and `cd` into the Defold directory. There should be a `Defold.exe` there. Now run the following command
```
packages\jre\bin\java -cp .\packages\defold-EDITORSHA.jar -Djogl.debug.DebugGL -Djogl.debug.TraceGL -Djogl.debug.GLContext.TraceSwitch -Djna.nosys=true -Ddefold.launcherpath=.\Defold -Ddefold.resourcespath=. -Ddefold.version=1.2.144 -Ddefold.editor.sha1=EDITORSHA -Ddefold.engine.sha1=ENGINESHA -Ddefold.buildtime=2019-01-08T13:38:03.220221 -Ddefold.channel=editor-alpha -Djava.ext.dirs=.\packages\jre\lib\ext -Djava.net.preferIPv4Stack=true -Dsun.net.client.defaultConnectTimeout=30000 -Dsun.net.client.defaultReadTimeout=30000 -Djogl.texture.notexrect=true -Dglass.accessible.force=false com.defold.editor.Start > debug.log 2>&1
```
This should produce a `debug.log` in the same directory. Please mail it to me, or add it to a comment here.

**Comment by NotLopX** [comment]
I still can't do it

**Comment by erikangelin-king** [comment]
Did the command above work? Can you mail me the debug.log file? erik.angelin@king.com

**Comment by NotLopX** [comment]
I don’t know how to type in the commands. Is there something I need to do for that

**Comment by erikangelin-king** [comment]
Hi! Well, I can't give you a complete tutorial on using the windows command line but a google search should surely fill in the gaps. If you have installed/unzipped Defold to your Desktop for instance the steps would be roughly the following:
* open the `Start`/`Windows` menu
* type `cmd`, this should find a program called `Command Prompt`
* `Enter` to run it
* This will open a command prompt with a saying something like
   ```Microsoft Windows [Version 10.0.17134.112]
   (c) 2018 Microsoft Corporation. All rights reserved.
   C:\>_
* Now type `cd Users` followed by `Enter`
* `cd Your.Username` (for instance `cd erik.angelin`) followed by `Enter`
* `cd Desktop` followed by `Enter`
* `cd Defold` followed by `Enter`
* Here, run the command in the earlier post with the SHA's replaced as described
* Exit the editor if it started
* `dir` will list the files in the `Defold` directory. There should be one called `debug.log`. You can also find it using explorer. Mail it to me and I'll have a look.

**Comment by NotLopX** [comment]
![capture](https://user-images.githubusercontent.com/16010495/51867189-73c12f00-2310-11e9-8a06-3cebf9b42e7d.PNG)

**Comment by NotLopX** [comment]
This was in the config file
`[bootstrap]
resourcespath = 
launcherpath = 

[build]
version = 1.2.145
editor_sha1 = c043948fbf2c57e88d4ca921ea9ab135081bb7e7
engine_sha1 = 9f376d88408bad9088a4187d4f874ae4767a0185
time = 2019-01-28T15:36:42.186253
channel = editor-alpha

[launcher]
jre = ${bootstrap.resourcespath}/packages/jre
java = ${launcher.jre}/bin/java
jar = ${bootstrap.resourcespath}/packages/defold-${build.editor_sha1}.jar
main = com.defold.editor.Start
debug = 1
vmargs = -Djna.nosys=true,-Ddefold.launcherpath=${bootstrap.launcherpath},-Ddefold.resourcespath=${bootstrap.resourcespath},-Ddefold.version=${build.version},-Ddefold.editor.sha1=${build.editor_sha1},-Ddefold.engine.sha1=${build.engine_sha1},-Ddefold.buildtime=${build.time},-Ddefold.channel=${build.channel},-Djava.ext.dirs=${launcher.jre}/lib/ext,-Djava.net.preferIPv4Stack=true,-Dsun.net.client.defaultConnectTimeout=30000,-Dsun.net.client.defaultReadTimeout=30000,-Djogl.texture.notexrect=true,-Dglass.accessible.force=false

[platform]
osx = -Xdock:icon=${bootstrap.resourcespath}/logo.icns,-Xdock:name=Defold
windows = 
linux = 

`

**Comment by NotLopX** [comment]
[debug.log](https://github.com/defold/editor2-issues/files/2805398/debug.log)

**Comment by NotLopX** [comment]
This version of C:\Users\Nic\OneDrive\Desktop\Defold\packages\jre\bin\java.exe is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher.

**Comment by erikangelin-king** [comment]
@NotLopX Ah, are you running on a 32 bit machine? You can check by right clicking on the `Start` / `Windows` menu, `Settings`, `About` and then under `Device specifications` it says `System type` and whatever bitness of operating system & processor you have.

**Comment by erikangelin-king** [comment]
That explains it, the patched java.exe I linked to is for 64 bit windows. I strongly recommend you run 64 bit windows on a 64 bit machine. We'll see if we can find time to patch a 32 bit java.exe for you, but likely not today.

**Comment by NotLopX** [comment]
Anything?

**Comment by NotLopX** [comment]
Error: Could not find or load main class com.defold.editor.Start

**Comment by erikangelin-king** [comment]
Huh. Weird. What if you reinstall the 32 bit version of the editor and just replace the java.exe?
As a heads up: We will in a near future drop editor support for 32 bit windows.
